src/main/java/org/veupathdb/lib/container/jaxrs/server/Server.java

```diff
@@ -44,6 +45,8 @@ public Server() {
     if (instance != null)
       throw new IllegalStateException(ERR_MULTI_SERVER);
 
+    PrometheusJVM.enable();
```

Comment:
If you just do this, is there a way to disable them?   I guess I'm confused by the PR title- maybe should be "Always enable jvm stats"- which is probably fine right?  Any reason to not enable?